chatsky/messengers/telegram/abstract.py

```diff
@@ -627,7 +627,7 @@ async def _on_event(self, update: Update, _: Any, create_message: Callable[[Upda
         data_available = update.message is not None or update.callback_query is not None
         if update.effective_chat is not None and data_available:
             message = create_message(update)
-            message.original_message = update
+            message.original_message = update.to_dict(recursive=True)
```

Comment:
This also needs to apply to extra fields in `Attachment`s.
Add a validator for `Attachment` and `Message` extras that modifies the extra field via `to_dict` if the field is of the `TelegramObject` value.

AFAIK if the extra field value is a dictionary from `to_dict` it should still work for the tg bot methods.